### PR TITLE
Test examples files in configmap directory

### PR DESCRIPTION
**Pull request by DanyC97**
Test the configMap examples added in https://github.com/kubernetes/website/pull/13078

### CODE REVIEW

**Comment by DanyC97** [comment]
@tengqm you fear about the cache PR seems to be a valid one.

I made a typo in the filename, Travis run and failed, then i've corrected the file using forced-pushed and now Travis still use the old file.

Any thoughts how i can force invalidate the file ?  

@zparnold fyi

**Comment by DanyC97** [comment]
@tengqm any chance you might know what is going on here ? 

the error says 

```
examples_test.go:592: configmap/configmaps.yaml: number of expected types (1) doesn't match number of docs in YAML (2)
examples_test.go:620: Directory configmap: Expected 2 examples, Got 1
```

but we got 2 examples no ?
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: special-config
  namespace: default
data:
  special.how: very
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: env-config
  namespace: default
data:
  log_level: INFO

```

**Comment by DanyC97** [comment]
digging a bit more i can see that we [parse](https://github.com/kubernetes/website/blob/master/content/en/examples/examples_test.go#L264) the yaml file and then the above error is due to [this logic](https://github.com/kubernetes/website/blob/master/content/en/examples/examples_test.go#L586-L589)

File: content/en/examples/examples_test.go [hunk 1]
```diff
@@ -404,6 +404,10 @@ func TestExampleObjectSchemas(t *testing.T) {
 		"application/zookeeper": {
 			"zookeeper": {&api.Service{}, &api.Service{}, &policy.PodDisruptionBudget{}, &apps.StatefulSet{}},
 		},
+		"configmap": {
+			"configmaps":            {&api.ConfigMap{}},
+			"configmap-multikeys":   {&api.ConfigMap{}, &api.ConfigMap{}},
```
Comment (tengqm) [inline content/en/examples/examples_test.go]: This is funny, you have configmaps.yaml listing two `ConfigMap` resources and you are telling the test cases that the `configmap-multikeys` has two `ConfigMap`s in it.

Comment (DanyC97) [reply content/en/examples/examples_test.go]: yes and i was expecting to work but yet, no dice. Am i missing something @tengqm  ?

Same thing in [here](https://github.com/kubernetes/website/blob/master/content/en/examples/examples_test.go#L395) works

Comment (DanyC97) [reply content/en/examples/examples_test.go]: note even with `"configmap-multikeys":   {&api.ConfigMap{}},` it fails

**Comment by DanyC97** [comment]
once https://github.com/kubernetes/website/pull/13145 is merged, rebase and carry on

**Comment by DanyC97** [comment]
ready for review @cody-clark , thanks !

**Comment by DanyC97** [comment]
as always thank you very much @tengqm

**Pull request merged**